_maps_2024/16-Trout.md

```diff
@@ -0,0 +1,10 @@
+---
+day: 16
+creator: Macy Trout
+title: Chloropleth
```

Comment:
@macytrout Just FYI, it's actually "choropleth" (i.e., no "L" after the "CH"). I used to make that mistake all the time.

```suggestion
title: Choropleth
```